delivery_transsmart/models/product_template.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models, _
+from openerp.exceptions import ValidationError
+
+
+class ProductTemplate(models.Model):
+    """
+    This model is used to hold the package types.
+    https://devdocs.transsmart.com/#_packages_retrieval
+    """
+    _inherit = 'product.template'
+
+    package = fields.Boolean()
```

Comment:
I don't think those fields should appear lile that as their names could conflict with other installed modules...

Even it's on v8, is it possible to think as generic as possible ? IMHO those fileds should maybe added in a more generic module (functional oriented one(s)).